### PR TITLE
refactor: use new_createPages in examples

### PR DESCRIPTION
**Pull request by tylersayshi**
Moving to new_createPages

I left `createPages` in the docs since we will drop the existing `createPages` soon and replace it with `new_createPages` under the same name.

### CODE REVIEW

**Comment by tylersayshi** [comment]
with the tests failing I'm seeing new_defineRouter needs to be mocked

**Comment by tylersayshi** [comment]
I am currently thinking through how best to test `renderRoute` in place of `getComponent`

https://github.com/dai-shi/waku/blob/main/packages/waku/tests/create-pages.test.ts#L531

also, in place of the skip tests: we should probably eventually add tests for the new define router.

File: packages/waku/src/router/fs-router.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { unstable_getPlatformObject } from '../server.js';
-import { createPages } from './create-pages.js';
+import { new_createPages as createPages } from './create-pages.js';
```
Comment (tylersayshi) [inline packages/waku/src/router/fs-router.ts:2]: @dai-shi did you want to wait before switching fs-router over? or are we ready?

**Comment by tylersayshi** [comment]
Looks like something must be broken

image

marking /draft

File: packages/waku/src/router/fs-router.ts (hunk 1 above)
Comment (dai-shi) [reply packages/waku/src/router/fs-router.ts:2]: I was thinking of migrating step by step, but if you are super confident, we can do altogether.

**Comment by tylersayshi** [comment]
react tweet example seems fine

weave render does not render anything when navigating away from, then back to home 🤔

File: examples/43_weave-render/src/components/BarLayout.tsx [hunk 2]
```diff
@@ -41,11 +41,11 @@ const BarLayout = ({ children }: { children: ReactNode }) => {
...
         <li>
           <Link
-            to={'/nested/bar' as never}
+            to={'/bar'}
             pending={<Pending isPending />}
             notPending={<Pending isPending={false} />}
           >
-            Nested / Bar
+            Bar
           </Link>
```
[the remark this answers is not in the document]
Comment (dai-shi) [reply examples/43_weave-render/src/components/BarLayout.tsx:49]: I'm not sure your original intent, but I thought it's an intentional 404 link.

[the remark this answers is not in the document]
Comment (tylersayshi) [reply examples/43_weave-render/src/components/BarLayout.tsx]: changed the link back but changed the text to make it obvious that link will 404 if that's ok with you

**Pull request merged**